### ISSUE TITLE
Add `/` behind all endpoints

### ISSUE THREAD

**Issue by mmacata** [mundialis/actinia_core#246]
Currently a `/` behind the endpoint leads to a 404. Example:
* `https://actinia.mundialis.de/api/v1/version` works
* `https://actinia.mundialis.de/api/v1/version/` leads to 404

Apparently this can be fixed so that both styles are recognized by adding a `/` behind the endpoint definition in `endpoints.py`

**Comment by anikaweinmann** [comment]
When I just tested it, it didn't work with `curl` though and the browser just kept appending `/` to the back of it.

**Comment by juleshaas** [comment]
Yes, the browser redirects `.../endpoint` to `.../endpoint/`, with curl however it only works with the `/` then. Would that still be an option?

**Comment by mmacata** [comment]
Then a wrapper for the endpoints or something like that would be nice. I don't think it is a good idea to change it if it doesn't work for both.
